### PR TITLE
Use RpoRandomCoin from crypto crate

### PR DESCRIPTION
**Pull request by Al-Kindi-0**
## Checklist before requesting a review
- Repo forked and branch created from `next` according to naming convention.
- Commit messages and codestyle follow [conventions](./CONTRIBUTING.md).
- Relevant issues are linked in the PR description.
- Tests added for new functionality.
- Documentation/comments updated according to changes.

### CODE REVIEW

File: core/src/lib.rs [hunk 1]
```diff
@@ -26,7 +26,10 @@ pub mod crypto {
     }
 
     pub mod random {
-        pub use crate::random::*;
+        pub use miden_crypto::rand::RpoRandomCoin;
+        pub use winter_crypto::{
+            DefaultRandomCoin as WinterRandomCoin, RandomCoin, RandomCoinError,
+        };
```
Comment (bobbinth) [inline core/src/lib.rs:32]: Should we pull these through `miden-crypto` as well?

**Review by bobbinth — COMMENTED** [review]
Looks good! Thank you! I left one comment inline. I think if we pull these imports through `miden-crypto`, we might be able get rid of `winter-crypto` dependency here altogether.

File: core/src/lib.rs (hunk 1 above)
Comment (Al-Kindi-0) [reply core/src/lib.rs:32]: Makes sense! I have re-exported the default random coin in a PR against the crypto crate.

Comment (bobbinth) [reply core/src/lib.rs:32]: Thank you! https://github.com/0xPolygonMiden/crypto/pull/241 is now merged!

**Review by bobbinth — APPROVED** [review]
All looks good! Thank you!

**Pull request merged**